andela/hc-leo
# Ft allocate different stakeholders 157694803

**What does this PR do?**
This PR adds checks allocation to different stakeholders. 

**Description of Task to be completed?**
Initially, a user who has team access to another user's healthchecks account is notified on the status of all the jobs defined in that account. Therefore, this feature enables user granting team access to their account to be able to assign different jobs to the person they are granting access to.

**How should this be manually tested?**
- Clone the repo, ```cd``` into the project and run ```./manage.py runserver```. 
- Navigate to the URL provided by the development server (usually ```localhost```) using a browser.
- Create an account or log in
- Create a new check and with a suitable ```name```,  ```period```, ```grace``` and ```nag```
- Navigate to ```Account Settings``` at the Navigation bar.
- Click ```Invite a Team Member``` button.
- Insert team member email and select a check.
- Click ```Send Invite```.
- When the invited team member accepts the invitation, they should only have access to their allocated checks

**What are relevant pivotal tracker stories?**
[#157694803](https://www.pivotaltracker.com/story/show/157694803)

**Screenshots**
<img width="1196" alt="screen shot 2018-06-20 at 10 23 23" src="https://user-images.githubusercontent.com/24792513/41643543-27d1566a-7474-11e8-92f9-3b5817e09289.png">
<img width="1235" alt="screen shot 2018-06-20 at 10 23 42" src="https://user-images.githubusercontent.com/24792513/41643556-30c894e0-7474-11e8-8cf3-53fcf9a96a66.png">
<img width="1202" alt="screen shot 2018-06-20 at 10 34 04" src="https://user-images.githubusercontent.com/24792513/41643986-7e0bb9de-7475-11e8-832e-2058557a5b8f.png">
